DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -408,6 +408,10 @@ has_detector FT0 && export ARGS_EXTRA_PROCESS_o2_tpcits_match_workflow="$ARGS_EX
 export ARGS_EXTRA_PROCESS_o2_tof_matcher_workflow="$ARGS_EXTRA_PROCESS_o2_tof_matcher_workflow --output-type matching-info,calib-info --enable-dia --use-fit"
 export CONFIG_EXTRA_PROCESS_o2_tof_matcher_workflow+=";$ITSEXTRAERR;$TRACKTUNETPC;$VDRIFTPARAMOPTION;"
 
+if [ $ALIEN_JDL_LPMPASSNAME == "cpass0" ]; then
```

Comment:
Ciao @noferini , 
Do you want this only for cpass0? Otherwise you should use something like:
```
if [[ -n $ALIEN_JDL_LPMCPASSMODE ]] && [[ $ALIEN_JDL_LPMCPASSMODE != "-1" ]]; then
  SETTING_ROOT_OUTPUT+="ENABLE_ROOT_OUTPUT_o2_tof_matcher_workflow= "
fi
```

Reply:
Note also that we use `[[ ]]`, instead of single brackets.

Reply:
I would keep only for cpass0
For any physics pass I would prefer to get empty matching histos in case of problems

Reply:
Let's assume for the moment that extra tolerance are needed only for cpass0 (cpass1 should be in any case better for TPC calibs)
 "[" -> "[[" DONE